dev/webpack-loaders/svg-loader/index.js

```diff
@@ -1,19 +1,18 @@
 const path = require('path');
 
-module.exports = function svgLoader(content) {
-    const match = content.match(/<svg([^>]+)+>([\s\S]+)<\/svg>/i);
-    const prefix = 'inline-';
-    const imageType = path
-        .dirname(this.resourcePath)
-        .split('/')
-        .pop();
-    const fileName = path.basename(this.resourcePath, '.svg');
-    const svg = match ? match[0].replace(/\n/g, ' ').trim() : '';
-    const markup = `<span class="${prefix}${fileName} ${
-        imageType !== '' ? prefix + imageType : ''
-    }">${svg}</span>`;
+function svgLoader(content) {
+	const match = content.match(/<svg([^>]+)+>([\s\S]+)<\/svg>/i);
+	const prefix = 'inline-';
+	const imageType = path.dirname(this.resourcePath).split('/').pop();
+	const fileName = path.basename(this.resourcePath, '.svg');
+	const svg = match ? match[0].replace(/\n/g, ' ').trim() : '';
+	const markup = `<span class="${prefix}${fileName} ${
+		imageType !== '' ? prefix + imageType : ''
+	}">${svg}</span>`;
 
-    this.value = markup;
+	this.value = markup;
 
-    return `module.exports = ${JSON.stringify({ markup })}`;
+	return `module.exports = ${JSON.stringify({ markup })}`;
 };
+
+  module.exports = svgLoader
```

Comment:
forgot to run prettier!

```suggestion
module.exports = svgLoader
```